files/file.py

```diff
@@ -62,10 +62,12 @@
         directories will be recursively deleted, and files or symlinks will be unlinked.
         If C(touch) (new in 1.4), an empty file will be created if the C(path) does not
         exist, while an existing file or directory will receive updated file access and
-        modification times (similar to the way `touch` works from the command line).
+        modification times (similar to the way `touch` works from the command line).  If
+        C(present), the file will be created if it does not exist.  If it does exist, no
```

Comment:
"the file will only be created if it does not exist. Otherwise, the modification time will not be changed."